docs/first-order/current_source_density.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<p>It might be nice to add a imshow plot here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>